examples/simple.js

```diff
@@ -132,31 +140,41 @@ d3.tsv("../data/PCA_transcript_expression_TMM_RPKM_log2_sample_data.6932.tsv", f
       data.forEach(function(d) {
         d.PC1 = +d.PC1;
         d.PC2 = +d.PC2;
+        d.PC3 = +d.PC3;
```

Comment:
Then I think I need to move this piece of code to index.js. Because in simple.js I only read tsv files once and draw the default graph.